_layouts/download.html

```diff
@@ -16,52 +16,100 @@ <h1>{{page.title}}</h1>
 
 {% comment %}Specific content from child layouts{% endcomment %}
 
+{% comment %}Compute the current scala release, default to Scala 2 version{% endcomment %}
+{% assign currentScalaRelease = site.scalaversion %}
+{% for release in site.data.scala-releases %}
+  {% if release.category == 'current_version' %}
+    {% assign releaseMajor = release.version | truncate:1, '' %}
+    {% if releaseMajor == '3' %}
+      {% comment %}found latest Scala 3 release{% endcomment %}
+      {% assign currentScalaRelease = release.version %}
+    {% endif %}
+  {% endif %}
+{% endfor %}
```

Comment:
Wouldn’t it be simpler to have a variable `site.scala3version` instead?

Reply:
that would also be sensible, I didn't originally just as would add another step for the release process

Reply:
OK.